### DISCUSSION TITLE
Question about the term 'Frameforks' in Readme

### DISCUSSION THREAD

**Discussion by minhsu12113** [cawa-93/vite-electron-builder#966] [General]
Hi Cawa, I have read your documentation on Readme section How it works and noticed that the term 'Frameforks' is written instead of 'Frameworks.' Is this a spelling mistake or a deliberate choice?

**Comment by cawa-93** [comment]
Nope. It's spelling mistake. Feel free to fix it

**Reply by cawa-93** [reply]
Moved discussion to related issue: https://github.com/cawa-93/vite-electron-builder/issues/969
